### DISCUSSION TITLE
“Something went wrong” when trying to view pull request

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#4199] [Mobile]
I can open <https://github.com/microsoft/terminal/pull/8114> in Safari just fine, but when I try to move from there to the GitHub 1.18.0 app, it says: "Something went wrong. Try again"

The pull request is "from unknown repository" and created by "ghost". Perhaps one of those triggers the problem.

**Comment by alcere** [comment]
Yikes! We are sorry for the trouble! I've open an issue to get this fixed as soon as possible. Thanks for reporting!

**Comment by alcere** [comment]
@KalleOlaviNiemitalo, the mobile app works slightly differently from the web version. The resource you are trying to access is hosted in an organization that requires SAML authentication that you are not part of. In the GitHub web version, we allow some content to be visible for users who don't belong to an organization, but in mobile, we don't for security reasons. 
We are sorry about this; we will try to improve the experience in the future.

**Reply by KalleOlaviNiemitalo** [reply]
Then, why am I able to view other pull requests of the microsoft/terminal repository in the mobile app? For example, <https://github.com/microsoft/terminal/pull/14> opens fine in the mobile app. Is this because of the source repository of the pull request?

If you have decided not to allow viewing <https://github.com/microsoft/terminal/pull/8114> in the mobile app without SAML authentication, then I would certainly appreciate a more precise error message. It is frustrating when I don't know whether the error is caused by a broken Internet connection or by a non-obvious GitHub access restriction.
